src/app/dialogs/import-experiment/import-experiment.component.html

```diff
@@ -30,6 +30,7 @@ <h1 mat-dialog-title>Import Experiment</h1>
     </mat-dialog-content>
     <div class="dialog-actions" mat-dialog-actions>
         <button mat-button (click)="onCancel()">Cancel</button>
-        <button mat-button (click)="import()" [disabled]="!addedFile || uploading">Import Experiment</button>
+        <button mat-button (click)="import()" [disabled]="!addedFile || polling!= 'NOT_STARTED'">Import
```

Comment:
Make sure, that this dialog is not dismissable by accidentally clicking on the background. This may impact an ongoing import. (or provide the import result outside of a dialog)